### ISSUE TITLE
Diff head/change text visibility

### ISSUE THREAD

**Issue by iquiw** [akabekobeko/redmine-theme-minimalflat2#93]
Another visibility issue.

In `diff` block, `head` and `change` texts are invisible as below.
![diffbeforechange](https://cloud.githubusercontent.com/assets/238681/23537737/848c674c-0011-11e7-8c1d-999512ece49d.png)

After the patch below is applied,
![diffafterchange](https://cloud.githubusercontent.com/assets/238681/23537741/85f4ef8c-0011-11e7-89ba-e4b8502129d6.png)

I have not prepared PR since I thought different colors might be preferable.

``` diff
diff --git a/src/stylus/App.styl b/src/stylus/App.styl
index 82aee85..9361fd5 100755
--- a/src/stylus/App.styl
+++ b/src/stylus/App.styl
@@ -3678,7 +3678,7 @@ h2 img {
   }

   .change {
-    color color_syntax_blue
+    color color_syntax_blue_light
     background: color_syntax_blue

     .change {
@@ -3691,11 +3691,11 @@ h2 img {
     background color_syntax_purple

     .filename {
-      color color_syntax_white
+      color color_syntax_base
     }

     .head {
-      color #DAACDF
+      color color_syntax_base
     }
   }
```

**Comment by akabekobeko** [comment]
Thanks for the report. I fixed the problem.

When Redmine v3.4 is released, v1.3.0 of this theme is also scheduled to be released. Check this because v1.3.0 will reflect this fix.

**Comment by akabekobeko** [comment]
If Redmine v3.4 is not released in March, it may release minimalflat2 v1.2.3 which fixed this bug etc in the first place.

**Comment by iquiw** [comment]
Thanks!
